src/neo/Network/P2P/TaskManager.cs

```diff
@@ -95,7 +95,7 @@ private void OnNewTasks(InvPayload payload)
             if (!sessions.TryGetValue(Sender, out TaskSession session))
                 return;
             // Do not accept payload of type InventoryType.TX if not synced on best known HeaderHeight
-            if (payload.Type == InventoryType.TX && Blockchain.Singleton.Height < sessions.Values.Max(p => p.LastBlockIndex))
+            if (payload.Type == InventoryType.TX && sessions.Values.Where(p => p.LastBlockIndex > Blockchain.Singleton.Height + 12).Count() > sessions.Count / 2)
```

Comment:
```suggestion
            if (payload.Type == InventoryType.TX && sessions.Values.Any(p => p.LastBlockIndex > Blockchain.Singleton.Height + 12))
```

What do you think?

Reply:
No, it can be easily attacked.

Reply:
I think `sessions.Count / 2` is too large.

Reply:
Adjusted to `sessions.Count/3`.

Reply:
So as it is now with +12 it will accept more transactions? Also based on the number of peers it has?

Reply:
Before this, some seed nodes can't relay transactions to consensus nodes in preview4 testnet.